crates/subspace-networking/src/create.rs

```diff
@@ -362,41 +304,19 @@ where
         // Create final structs
         let (command_sender, command_receiver) = mpsc::channel(1);
 
-        let shared = Arc::new(Shared::new(local_peer_id, command_sender));
+        let kademlia_tasks_semaphore =
+            Arc::new(ResizableSemaphore::new(KADEMLIA_BASE_CONCURRENT_TASKS));
+        let regular_tasks_semaphore =
+            Arc::new(ResizableSemaphore::new(REGULAR_BASE_CONCURRENT_TASKS));
```

Comment:
New semaphores are clonable, there is no need to wrap them in an additional `Arc` anymore.